src/ui-fileops.cc

```diff
@@ -583,6 +583,9 @@ gboolean copy_file(const gchar *s, const gchar *t)
 			}
 		}
 
+	fclose(fi); fi = nullptr;
```

Comment:
Could you please add comment why we explicitly close files here?

Reply:
Good point, let me know if the level of details is suitable.

Reply:
LGTM